lib/python/ensembl/brc4/runnable/dump_stable_ids.py

```diff
@@ -215,6 +216,19 @@ def _name_event(self) -> None:
         else:
             raise UnsupportedEvent(f"Event {self.from_set} to {self.to_set} is not supported")
     
+    def clean_pairs(self) -> None:
+        """Remove the empty old pairs when the event is not 'new'."""
+        if not self.name:
+            self._name_event()
+        
+        if self.name != "new":
+            new_pairs = []
+            for pair in self.pairs:
+                if pair["old_id"] == "":
+                    continue
+                new_pairs.append(pair)
```

Comment:
This is the only way I can think that justifies the need of `continue`:
```suggestion
                if ("old_id" in pair) and (pair["old_id"] == ""):
                    continue
                new_pairs.append(pair)
```

Reply:
Replaced with .get()